src/trigrams/cli.py

```diff
@@ -30,7 +30,7 @@ def main():
     options = parser.parse_args()
 
     with open(options.input_file, "r") as fin:
-        input_data = fin.read().replace("\n", "")
+        input_data = fin.read().replace("\n", " ")
```

Comment:
Nice! Thanks for spotting a bug! Would be great with a test to check this behavior though.